packages/transformer/src/test/IModelTransformerUtils.ts

```diff
@@ -302,6 +308,10 @@ export async function assertIdentityTransformation(
             mappedRelationTargetInTargetId
           );
         } else if (!propChangesAllowed) {
+          if (ignoreFedGuidsOnAlwaysPresentElementIds && propName === "federationGuid" && alwaysPresentElementIds.has(sourceElemId)) {
+            // Skip comparing this prop if its the fedguid and its one of the always present element ids and ignoreFedGuidsOnAlwaysPresentElementIds is true.
```

Comment:
```suggestion
            // Skip comparing this prop if its the fedguid and its one of the root elements and ignoreFedGuidsOnAlwaysPresentElementIds is true.
```

Reply:
actually remove this if and add this condition to the `propChangesAllowed` boolean variable

Reply:
done